willa/config/__init__.py

```diff
@@ -60,3 +60,11 @@ def get_lance() -> LanceDB:
     """Return a configured instance of a LanceDB class."""
     embeddings = OllamaEmbeddings(model='nomic-embed-text', base_url=CONFIG['OLLAMA_URL'])
     return LanceDB(embedding=embeddings, uri=CONFIG['LANCEDB_URI'], table_name='willa')
+
+def get_ollama() -> ChatOllama:
+    """Return a configured instance of a ChatOllama chat model."""
+    return ChatOllama(
+                model=CONFIG['CHAT_MODEL'],
+                temperature=float(CONFIG['CHAT_TEMPERATURE']),
+                base_url=CONFIG['OLLAMA_URL']
+            )
```

Comment:
The only thing I could say is maybe this should be `get_model` and be declared to return a `BaseChatModel`.  That way, we don't have to change anything externally when we move off Ollama.

In fact, we could even make it configurable as well - to use a local Ollama model or a remote Bedrock model - if we did it that way.